SUMMARY.md

```diff
@@ -28,10 +28,9 @@
 
 ## Tutorials on EthStorage early testnet
 
-* [Upload Your First File/Folder with ethfs-uploader](tutorials/upload-your-first-file-folder-with-ethfs-uploader.md)
-* [Upload Your First File with ethfs-sdk](tutorials/upload-your-first-file-with-ethfs-sdk.md)
+* [Upload Your First File/Folder with ethfs-cli](tutorials/upload-your-first-file-folder-with-ethfs-cli.md)
```

Comment:
You may need to double check if the change can take effect after this PR is merged in the docs.web3url.io